src/wr_logic_ai/nodes/coord_calculations.py

```diff
@@ -0,0 +1,76 @@
+import math
+
+class coord_calculations:
+    #def __init__(self, )
+    earth_radius = 6378100 # in meters
+
+    @staticmethod
+    def get_coords(lat1, lon1, distance, num_vertices):
+        orig_dist = distance
+        coords = []
+        bearing = 0
+        mult = 0
+
+        coords.append({'lat': lat1, 'long': lon1})#, 'distance': orig_dist, 'bearing': bearing})
+
+        for i in range(num_vertices):
+            # update the distance as needed
+            if (i > 1) and (i % 2) == 0:
+                mult += orig_dist
+            coords.append(coord_calculations.calc_dest_coord(coords[i]['lat'], coords[i]['long'], distance + mult, bearing))
+            bearing += 90
+
+        return coords
+
+    @staticmethod
+    def calc_dest_coord(lat1, lon1, distance, bearing):
+        lat2_rad = 0
```

Comment:
The variable names `lat1`, `lat2`, etc used in this class are ambiguous. I'm getting the idea that one is the origin and the other is the destination, but it would be better if they were written out.

Reply:
Fixed in 28b2f711.